Documentation/building/cross-building.md

```diff
@@ -27,9 +36,34 @@ and if you wanted to generate the rootfs elsewhere:
 
     ben@ubuntu ~/git/coreclr/ $ sudo ROOTFS_DIR=/home/ben/coreclr-cross/arm ./build-rootfs.sh arm
 
+Patching Urcu
```

Comment:
Whilst this is an issue, would it perhaps be cleaner to link to the newer version of the generic.h file https://github.com/urcu/userspace-rcu/blob/v0.7.15/urcu/uatomic/generic.h